package.json

```diff
@@ -18,6 +19,7 @@
   },
   "homepage": "https://github.com/glynnw/FCC-Discord-Bot#readme",
   "dependencies": {
+    "chalk": "^2.4.1",
```

Comment:
Have you considered using the `colors` dependency, since we already use it in other files?